src/main/java/io/jenkins/plugins/artifact_manager_s3/S3BlobStore.java

```diff
@@ -82,6 +82,13 @@
     private static String PREFIX = System.getenv("S3_DIR");
     @SuppressWarnings("FieldMayBeFinal")
     private static String REGION = System.getProperty(S3BlobStore.class.getName() + ".region");
+    @SuppressWarnings("FieldMayBeFinal")
+    private static boolean DEFAULT_DELETE_BLOBS = Boolean.getBoolean(S3BlobStore.class.getName() + ".defaultDeleteBlobs");
```

Comment:
Get rid of the `default` prefix, which makes no sense from a user PoV, and make the instance variables `transient` since we do not care for them to appear in the serial form unless this is switched to explicit configuration.